h/views/badge.py

```diff
@@ -70,6 +70,14 @@ def badge(request):
         cache_control.public = True
         cache_control.max_age = 86400  # 1 day
 
+        # `pyramid_authsanity` sets a response callback which adds Vary=Cookie
+        # which will totally break our caching. To get around this we add
+        # another callback which should be called after to dsiable it.
+        def disable_vary_header(_request, response):
+            response.vary = None
+
+        request.add_response_callback(disable_vary_header)
```

Comment:
I can't test this locally, as I don't know how to prompt the situation, but it seems to be 100% in QA... WHY. I've run the docker container locally as it is run in QA as far as I can tell, but the behavior is different.